scripts/build_tizen.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+##############################################################################
+#  Example command to build the Tizen target.
+##############################################################################
+# 
+# This script shows how one can build a Caffe2 binary for raspbian. The build
+# is essentially much similar to a host build, with one additional change
+# which is to specify -mfpu=neon for optimized speed.
+
+
+# Download caffe2 source code
+# If you do not have a Tizen device, Please, run qemu-arm based chroot command. 
+# $ sudo chroot ~/tizen-rootfs qemu-arm-static /usr/bin/bash 
+git clone --recursive https://github.com/caffe2/caffe2.git && cd caffe2
```

Comment:
You wouldn't need this - if you have this script you should already have checked out the repository.